api/operator_license.go

```diff
@@ -100,6 +100,7 @@ func (op *Operator) LicensePut(license string, opts *WriteOptions) (*LicenseRepl
 	r := op.c.newRequest("PUT", "/v1/operator/license")
 	r.setWriteOptions(opts)
 	r.body = strings.NewReader(license)
+	r.header.Set("Content-Type", "application/octet-stream")
```

Comment:
Great job finding these exceptions! How did you find them out of interest? Did tests break for these endpoints because of the wrong content type?

I can't think of any other exceptions but then I doubt I'd have remembered this one either so wondering if there is anything else we should do to be sure we caught all the non-json endpoints.

If you already went through exhaustively though then great!